src/squlearn/kernel/ml/qsvr.py

```diff
@@ -9,13 +9,52 @@ class QSVR(SVR):
 
     This class is a wrapper of sklearn.svm.SVR. It uses a quantum kernel matrix
```

Comment:
Could you change this paragraph, so it wraps to less than 99 characters per line? Black doesn't do this for docstrings...